prompting/tools/datasets/wiki.py

```diff
@@ -316,14 +316,20 @@ def _random_date(self) -> str:
                 date_sentence = self.extract_dates_and_sentences(context['content'])
                 context['content'] = date_sentence[1]
                 context['extra']['date'] = date_sentence[0]
-                if context['content'] is None:
+                if not context['content']:
+                    continue
+                elif not context['extra']['date']:
                     continue
                 else:
                     return context
```

Comment:
Ignore for now, just a cosmetic suggestion
```suggestion
                if context.get("content") and context["extra"].get("date"):
                    return context
```